### PR TITLE
Census Regression sample for ML Toolbox

### PR DESCRIPTION
**Pull request by nikhilk**
End to End notebooks for local and service-based machine learning using Cloud Machine Learning Engine, BigQuery, Dataflow, and Datalab ML Toolbox.

### CODE REVIEW

File: samples/ML Toolbox/Regression/Census/4 Service Evaluate.ipynb [hunk 1]
```diff
@@ -222,87 +199,272 @@
...
    "cell_type": "code",
-   "execution_count": 8,
+   "execution_count": 10,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "%bq datasource --name eval_results --paths gs://cloud-ml-users-datalab-workspace/census/evaluation/predictions*\n",
+    "{\n",
```
Comment (qimingj) [inline samples/ML Toolbox/Regression/Census/4 Service Evaluate.ipynb]: Ideally, %bq datasource can optionally take a schema json file, which is already produced in batch prediction job.

File: samples/ML Toolbox/Regression/Census/4 Service Evaluate.ipynb [hunk 2]
```diff
@@ -222,87 +199,272 @@
...
+    "  \"schema\": [\n",
+    "    {\n",
+    "      \"type\": \"STRING\", \n",
+    "      \"mode\": \"nullable\", \n",
+    "      \"name\": \"SERIALNO\"\n",
+    "    }, \n",
+    "    {\n",
+    "      \"type\": \"FLOAT\", \n",
+    "      \"mode\": \"nullable\", \n",
+    "      \"name\": \"predicted_target\"\n",
```
Comment (qimingj) [inline samples/ML Toolbox/Regression/Census/4 Service Evaluate.ipynb]: This is called "predicted" now? And next one should be "target"? Do you want to re-execute the cell to get the latest output? Not a big deal though.

File: samples/ML Toolbox/Regression/Census/4 Service Evaluate.ipynb (hunk 1 above)
Comment (nikhilk) [inline samples/ML Toolbox/Regression/Census/4 Service Evaluate.ipynb]: Yes, this is something that @yebrahim will add support for.

File: samples/ML Toolbox/Regression/Census/4 Service Evaluate.ipynb (hunk 2 above)
Comment (nikhilk) [inline samples/ML Toolbox/Regression/Census/4 Service Evaluate.ipynb]: Good catch... need to fix in a subsequent change (given where we are now)

**Pull request merged**